qubes-bug-report

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env python3
+
+import subprocess as sub
```

Comment:
Please use just `import subprocess` - module aliases makes it harder to read the script